### PR TITLE
Improve bucket API

### PR DESCRIPTION
**Pull request by Frago9876543210**
## Introduction
Added method for obtaining kind of liquid

### API changes
Added `LiquidBucket::getLiquid()` method

### CODE REVIEW

File: src/event/player/PlayerBucketEvent.php [hunk 1]
```diff
@@ -62,7 +64,7 @@ public function __construct(Player $who, Block $blockClicked, int $blockFace, It
 	/**
 	 * Returns the bucket used in this event
 	 *
-	 * @return Item
+	 * @return LiquidBucket|Bucket|Item
```
Comment (dktapps) [inline src/event/player/PlayerBucketEvent.php]: This is orthogonal to adding liquid type API, and also unnecessary since `Item` includes these types.

Comment (Frago9876543210) [reply src/event/player/PlayerBucketEvent.php]: Yes, but how IDE understand that Item has getLiquid method?

Comment (dktapps) [reply src/event/player/PlayerBucketEvent.php]: `instanceof`, same as anything.

Comment (Frago9876543210) [reply src/event/player/PlayerBucketEvent.php]: Well, method called "getBucket" and should already it, not any item

**Review by dktapps — CHANGES_REQUESTED** [review]
There are two unrelated changes in this PR. Please remove one of them.
